cdb/tx.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2016 The Cockroach Authors.
```

Comment:
so @spencerkimball do we really want our go instructions to point to using this repo? I wouldn't add a dependency to a whole repo for one 10-line function. And this would also make the instructions for `Go` different than the instructions for any other language (unless you continue this spree and rip of all the wrappers into new repos).